<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin2</class>
 <widget class="QMainWindow" name="MainWin2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QLineEdit" name="LEditSendPort"/>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>数据载体</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkTimeStamp">
           <property name="text">
            <string>时间戳</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="LEditSendAddress"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>端口</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>地址</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>发送UDP窗口</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="btnSend">
         <property name="text">
          <string>发送UDP网路数据报</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="TEditSend"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>接收UDP窗口</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>数据载体</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReceiveLog">
           <property name="text">
            <string>历史记录</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="TEditReceive"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>信息来源</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="LEditReceiveAddress"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>监听端口</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="LEditReceivePort"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="btnReceiveEnable">
         <property name="text">
          <string>开启端口监听</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
